packages/maskbook/src/plugins/RedPacket/UI/RedPacketInList.tsx

```diff
@@ -75,3 +141,133 @@ export function RedPacketInList(props: RedPacketInListProps) {
         </ListItem>
     )
 }
+export interface RedPacketInHistoryListProps {
+    key: string
```

Comment:
IT'S NOT WORKING. `key` is a special prop name and will be removed by React.

Reply:
you're right, just wrap a `<div key={...} />` for `<RedPacketInHistoryList>` solves `react-jsx-dev-runtime.development.js?9b1f:115 Warning: Each child in a list should have a unique "key" prop.
`